### PR TITLE
Release Portworx services

### PR DESCRIPTION
**Pull request by disrani-px**

### CODE REVIEW

File: repo/packages/C/cassandra-portworx/0/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "description": "Apache Cassandra backed by Portworx volumes",
+  "framework": true,
+  "maintainer": "support@mesosphere.io",
+  "minDcosReleaseVersion": "1.8",
+  "name": "cassandra-portworx",
+  "packagingVersion": "3.0",
+  "postInstallNotes": "DC/OS Cassandra is being installed!\n\n\tDocumentation: https://docs.mesosphere.com/latest/usage/service-guides/cassandra\n\tIssues: https://docs.mesosphere.com/support/",
+  "postUninstallNotes": "DC/OS cassandra is being uninstalled.",
+  "preInstallNotes": "This DC/OS Service is currently a beta candidate undergoing testing as part of a formal beta test program.\n\nThere may be bugs, incomplete features, incorrect documentation, or other discrepancies.\n\nDefault configuration requires 3 agent nodes each with: 0.5 CPU | 4096 MB MEM | 1 10240 MB Disk\n\nContact Mesosphere before deploying this beta candidate service. Product support is available to approved participants in the beta test program.",
+  "selected": true,
```
Comment (joerg84) [inline repo/packages/C/cassandra-portworx/0/package.json]: Is this package selected?

Comment (ryadav88) [reply repo/packages/C/cassandra-portworx/0/package.json]: No

File: repo/packages/K/kafka-portworx/0/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "description": "Apache Kafka",
+  "framework": true,
+  "maintainer": "support@mesosphere.io",
```
Comment (ryadav88) [inline repo/packages/K/kafka-portworx/0/package.json]: `maintainer` should be a px email

File: repo/packages/K/kafka-portworx/0/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  "minDcosReleaseVersion": "1.8",
```
Comment (ryadav88) [inline repo/packages/K/kafka-portworx/0/package.json]: `minDcosReleaseVersion` should be min version of dc/os that px will support

File: repo/packages/K/kafka-portworx/0/package.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  "name": "kafka-portworx",
+  "packagingVersion": "3.0",
+  "postInstallNotes": "DC/OS Apache Kafka Service is being installed.\n\n\tDocumentation: https://docs.mesosphere.com/current/usage/service-guides/kafka\n\tIssues: https://dcosjira.atlassian.net/projects/KAFKA/issues",
+  "postUninstallNotes": "DC/OS Apache Kafka Service is being uninstalled.\nPlease follow the instructions at https://docs.mesosphere.com/current/usage/service-guides/kafka/uninstall to remove any persistent state if required.",
+  "preInstallNotes": "This DC/OS Service is currently a beta candidate undergoing testing as part of a formal beta test program.\n\nThere may be bugs, incomplete features, incorrect documentation, or other discrepancies.\n\nDefault configuration requires 3 agent nodes each with: 1.0 CPU | 2048 MB MEM | 1 5000 MB Disk\n\nContact Mesosphere before deploying this beta candidate service. Product support is available to approved participants in the beta test program.",
```
Comment (ryadav88) [inline repo/packages/K/kafka-portworx/0/package.json]: `preInstallNotes` remove the first line there about beta program and replace it with, `This DC/OS Service is currently in preview.`
also remove `Contact Mesosphere before deploying this beta candidate service. Product support is available to approved participants in the beta test program.`

File: repo/packages/K/kafka-portworx/0/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  "selected": true,
```
Comment (ryadav88) [inline repo/packages/K/kafka-portworx/0/package.json]: `selected` should be false

File: repo/packages/K/kafka-portworx/0/package.json (hunk 3 above)
Comment (triclambert) [reply repo/packages/K/kafka-portworx/0/package.json]: This should also be in sync with the Mesosphere SDK-based versions.  (All SDK packages will GA with 1.9.)

File: repo/packages/K/kafka-portworx/0/resource.json [hunk 6]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (ryadav88) [inline repo/packages/K/kafka-portworx/0/resource.json]: please move all these frameworks under P/portworx-*

File: repo/packages/K/kafka-portworx/0/package.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
+{
+  "description": "Apache Kafka",
+  "framework": true,
+  "maintainer": "support@mesosphere.io",
+  "minDcosReleaseVersion": "1.8",
+  "name": "kafka-portworx",
+  "packagingVersion": "3.0",
```
Comment (ryadav88) [inline repo/packages/K/kafka-portworx/0/package.json]: `packagingVersion` should be 4.0.  this is set in the release_builder script

**Comment by disrani-px** [comment]
@ryadav88 I have updated the packages with the feedback, please re-review.

**Pull request merged**